core/domain/state_domain.py

```diff
@@ -961,7 +965,11 @@ class RecordedVoiceovers(python_utils.OBJECT):
     """
 
     def __init__(self, voiceovers_mapping):
-        """Initializes a RecordedVoiceovers domain object."""
+        """Initializes a RecordedVoiceovers domain object.
+
+        Args:
+            voiceovers_mapping: Recorded Voiceovers domain object.
```

Comment:
Add the type of arg here. (Following the style guide.)

Reply:
Done